scripts/README.md

```diff
@@ -0,0 +1,52 @@
+# Setup & Build scripts
+
+This directory contains a few scripts to simplify setting up the Windows agent
+of webpagetest.
+
+The scripts have been developed and tested on Windows 8.1 Enterprise
```

Comment:
That would be great, thanks. Can you send me the details to access such an instance via email?